R/runner.R

```diff
@@ -78,6 +78,17 @@ runner_run <- function(key_report_id, key, root, name, parameters, instance,
     Sys.sleep(poll)
   }
 
+  ## For very fast report running we can get into the state that
```

Comment:
I think this belongs within the above loop already? So an `if (is.na(id)) next` or similar at ~74